pydrive2/fs/spec.py

```diff
@@ -418,11 +424,7 @@ def info(self, path):
     def ls(self, path, detail=False):
         bucket, base = self.split_path(path)
 
-        cached = base in self._ids_cache["dirs"]
-        if cached:
-            dir_ids = self._ids_cache["dirs"][base]
-        else:
-            dir_ids = self._path_to_item_ids(base)
+        dir_ids = self._path_to_item_ids(base)
```

Comment:
C: since now `self._path_to_item_ids(base)` also does caching it's not clear why we should have a separate check above. I need to review this one more time though. The same applies in the `find` below, and removal of the cache calls since we now call cache inside path_to_item_ids.